pkg/resources/requests.go

```diff
@@ -67,3 +91,18 @@ func ResourceQuantity(name corev1.ResourceName, v int64) resource.Quantity {
 		return *resource.NewQuantity(v, resource.DecimalSI)
 	}
 }
+
+func (req Requests) CountIn(capacity Requests) int32 {
+	var result *int32
+	for rName, rValue := range req {
+		capacity, found := capacity[rName]
+		if !found {
+			return 0
+		}
+		count := int32(capacity / rValue)
```

Comment:
What if those are not divisible?

Reply:
Then we will not be able to fit another full pod, which is what we wont - this function is used to count the number of pods which can fully fit into the free capacity. 

Added a test case for it: https://github.com/kubernetes-sigs/kueue/pull/3256/files#diff-f07b44933bee01621ef7cc05b4a8c30a8962963044b725b600a2dee63846b727R74-R82

Reply:
That sounds reasonable. Thanks for pointing out the unit testing case!